modules/client-configuration/pages/clients-sleses.adoc

```diff
@@ -119,10 +119,10 @@ include::snippets/addchannels_vendor_webui.adoc[]
 
 endif::[]
 
-
-ifeval::[{uyuni-content} == true]
 The channels you need for this procedure are:
 
+ifeval::[{uyuni-suma} == true]
```

Comment:
Isn't this supposed to be `suma-content`?

Reply:
thanks for spotting!